### PR TITLE
Add tests for module

### PR DESCRIPTION
**Pull request by bart613**
This PR adds tests for module

Tests included:
- test_create_alb_listener_rule
- test_create_alb_listener_rule_custom_starting_priority
- test_create_alb_listener_rule_host_condition
- test_create_alb_listener_rule_two_paths

### CODE REVIEW

File: test/test_tf_alb_listener_rules.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
+import unittest
+import os
+from subprocess import check_call, check_output
+
+cwd = os.getcwd()
+
+
+class TestTFALBListenerRules(unittest.TestCase):
```
Comment (tomyan) [inline test/test_tf_alb_listener_rules.py:8]: Being very picky I don't like the abbreviation in the name. Is very quick to work out what's it short for, but still causes me to pause when looking at the code.

**Review by tomyan — APPROVED** [review]

File: test/test_tf_alb_listener_rules.py (hunk 1 above)
Comment (bart613) [reply test/test_tf_alb_listener_rules.py:8]: @tomyan: I don't like it either - any suggestions for what to replace it with?  Just `Test_tf_alb_listener_rules`?

Comment (grahamlyons) [reply test/test_tf_alb_listener_rules.py:8]: I wouldn't like the underscores in the class name... The current name is the lesser of two weevils.

**Review by grahamlyons — APPROVED** [review]

**Comment by tomyan** [comment]
This code has been largely replaced with tf_frontend_service_routing

**Pull request closed without merging**